app/views/volunteer_applications/new.html.slim

```diff
@@ -0,0 +1,8 @@
+.row
+  .col-xs-12
+    h1= t('volunteer_application')
+.row
+  .col-xs-12
+    = simple_form_for(@volunteer, url: volunteer_applications_path) do |f|
+      = render 'volunteers/form', f: f
+      = f.button :submit, t('submit_application'), class: 'btn btn-success'
```

Comment:
PS:
to make locale rebaseing easyer in the future sort the yaml files alphabetically......
with:
```shell
gem install i18n_yaml_sorter
```

and then you can do this in the shell:
```shell
sort_yaml < de.yml > de_sorted.yml
```